pkg/middleware/logging/logging.go

```diff
@@ -74,11 +74,12 @@ func (t *logTripper) Request(r *http.Request) (*http.Response, error) {
 	start := time.Now()
 	res, err := t.tripper.Request(r)
 	duration := time.Since(start)
-	fragment := viewproxy.FragmentFromContext(r.Context())
+	requestable := multiplexer.RequestableFromContext(r.Context())
```

Comment:
This was previously returning a `multiplexer.fragment`, not a `viewproxy.Fragment` so this should be equivalent